examples/platform-app/cypress.env.json

```diff
@@ -2,5 +2,7 @@
     "dhis2_username": "admin",
     "dhis2_password": "district",
     "dhis2_base_url": "http://localhost:8080",
-    "dhis2_datatest_prefix": "networkshim-platformapp"
+    "dhis2_datatest_prefix": "networkshim-platformapp",
+    "network": "live",
+    "dhis2_api_version": "37"
```

Comment:
Names can be bikeshedded, but I chose these because it's reasonably clear that we are talking about the dhis2 core api version here, and that isn't really a "minor", and shorter is a good thing, imo.

We may also want to extend the shim to handle e.g. `apphub_api_version` and I think that's fine. Not everything needs to be prefixed with dhis2.

I chose `network` because it's declarative and short, e.g. `network=capture`, `network=stub`, and `network=live` is pretty clear to me in terms of what they do with the network.

Reply:
I'm on board with these new names.

Reply:
I may have jumped over board. :|